ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -716,6 +716,37 @@ tests:
     - chain: ipi-install-heterogeneous-day2-kerneltype
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-public-ipv4-pool-arm-f7
```

Comment:
btw, suggest to add a z-stream job in openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.16.yaml together.

Reply:
Done.